test/parallel/test-repl-tab-complete.js

```diff
@@ -588,11 +620,29 @@ const testNonGlobal = repl.start({
   useGlobal: false
 });
 
-const builtins = [['Infinity', 'Int16Array', 'Int32Array',
-                   'Int8Array'], 'I'];
+const builtins = [
+  [
+    'if',
+    'import',
+    'in',
+    'instanceof',
+    '',
+    'Infinity',
+    'Int16Array',
+    'Int32Array',
+    'Int8Array',
+    'Intl',
```

Comment:
Optional nit: I find this syntax easier to parse as a human reader
```suggestion
    ...(common.hasIntl ? ['Intl'] : []),
```

Reply:
Yeah, much more readable, fixed!